openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py

```diff
@@ -66,14 +68,16 @@ def get_mean_and_stddevs(self, sites, rup, dists, imt, stddev_types):
         for spec of input and result values.
         """
         working_std_types = copy.copy(stddev_types)
-
+		
         if 'set_between_epsilon' in self.params:
             if (const.StdDev.INTER_EVENT not in
                     self.gmpe.DEFINED_FOR_STANDARD_DEVIATION_TYPES):
                 raise ValueError('The GMPE does not have between event std')
             working_std_types.append(const.StdDev.INTER_EVENT)
             working_std_types.append(const.StdDev.INTRA_EVENT)
             working_std_types.append(const.StdDev.TOTAL)
+        if 'apply_swiss_intensity_amplification' in self.params:
```

Comment:
I think we should also add a test that the 'original' model supports MMI

Reply:
Would adding a DEFINED_FOR_INTENSITY_MEASURE_TYPES = {MMI} do this? In any case, I guess that as constructed it could also work for other IMTs. 

However, in the (near) future we will need to apply period-dependent amplification factors to ground motion IMs, so maybe we could discuss how this can be extended (i.e. how to use period-specific amplfactor' variables).